.github/workflows/pull-request.yaml

```diff
@@ -75,7 +75,12 @@ jobs:
         uses: actions/setup-go@f111f3307d8850f501ac008e886eec1fd1932a34 # v5.3.0
         with:
           go-version: ${{ steps.go_version.outputs.go_version }}
-        if: matrix.os == 'darwin'
+        if: matrix.os != 'linux'
```

Comment:
We're no longer in the go image, so we'll need to get golang set up for the windows build, too.